src/witan/gwyn/utils.clj

```diff
@@ -11,3 +11,43 @@
   (if (zero? dd)
     0.0
     (/ ^double d dd)))
+
+(def unwanted-property-types
```

Comment:
Sorry to be a pain but I'd probably put these into `gwyn.clj` as they're definitely contextually relevant, where as 'utils' is usually for stuff that is irrelevant or generic.